plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/Session.kt

```diff
@@ -159,23 +170,35 @@ class Session(val tabID: String, val project: Project) {
             }
         }.onFailure { /* Noop on diff telemetry failure */ }
 
-        newFilePaths.forEach {
-            resolveAndCreateOrUpdateFile(selectedSourceFolder, it.zipFilePath, it.fileContent)
-            it.changeApplied = true
-        }
+        insertNewFiles(newFilePaths)
```

Comment:
Can we use consistent verbs here? And where are we inserting? These "new" files can be "created" or "updated". Should this be `createOrUpdateLocalFiles` and the below `deleteLocalFiles`?

Reply:
To make it consistent with repo `aws-toolkit-vscode`, I named it `insertNewFiles`.
![image](https://github.com/user-attachments/assets/84340aba-f52d-4e7d-a397-b638d170cbbf)